eng/pipelines/templates/jobs/ci.yml

```diff
@@ -268,7 +268,10 @@ jobs:
             parameters:
               PackageName: ${{artifact.name}}
               ServiceName: ${{parameters.ServiceDirectory}}
-              ForRelease: false
+              ${{ if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'), ne(variables['Skip.Release'], 'true')) }}:
```

Comment:
Since I already dislike the proliferation of this conditional, I think it would be better to declare a parameter `ForRelease` in `archetype-sdk-client.yml` and then set that parameter on this conditional instead. That way we contain usage of this conditional line in files where it's already used.

Reply:
Done!